schemachange/sc_drop_table.c

```diff
@@ -45,7 +45,7 @@ static int delete_table(struct db *db, void *trans)
     remove_constraint_pointers(db);
 
     int rc, bdberr;
-    if ((rc = bdb_close_only_tran(db->handle, trans, &bdberr))) {
+    if ((rc = bdb_close_only_tran(db->handle, NULL, &bdberr))) {
```

Comment:
It just closes the handle. I thought you said close doesn't need tran. And if the whole thing fails later, we will reopen.